ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/CV_PATHFINDER_TESTS.yml

```diff
@@ -82,7 +82,12 @@ default_node_types:
   - node_type: l3leaf
     match_hostnames:
       - ".*leaf.*"
-
+# Test p2p_uplinks_mtu
+p2p_uplinks_mtu: 9214
+platform_settings:
+  - platforms: [ default ]
+    # platform settings p2p_uplinks_mtu takes precedence
+    p2p_uplinks_mtu: 9194
```

Comment:
maybe pick a different value than the default one? for this it may be better to not mix and match inside CV_PATHFINDER tests and rather to add an entry in some different test device to make sure we dont lose it inside this big "example"